src/snippets/template.py.tmpl

```diff
@@ -121,30 +121,37 @@ def output_list():
       <%     } 
            }%>sys.stdout.write("\n")
       <% } else if (fileType === "fasta") { %>
-      if ("entries" in item):
-        for entry in item["entries"]:
-          for locations in entry["entry_protein_locations"]:
-            for fragment in locations["fragments"]:
-              start = fragment["start"]
-              end = fragment["end"]
-              sys.stdout.write(">" + item["metadata"]["accession"] + HEADER_SEPARATOR
-                               + entry["accession"] + HEADER_SEPARATOR
-                               + str(start) + "..." + str(end) + HEADER_SEPARATOR
-                               + item["metadata"]["name"] + "\n")
-              seq = item["extra_fields"]["sequence"]
-              fastaSeqFragments = [seq[0+i:LINE_LENGTH+i] for i in range(0, len(seq), LINE_LENGTH)]
-              for fastaSeqFragment in fastaSeqFragments:
-                sys.stdout.write(fastaSeqFragment + "\n")
+      entries = None
+      if ("entry_subset" in item):
+        entries = item["entry_subset"]
+      elif ("entries" in item):
+        entries = item["entries"]
+      
+      if entries is not None:
+        entries_header = ""
+        for entry in entries:
+          entries_header += " " + entry["accession"] + "(" + ";".join(
```

Comment:
The leading space before the entry accession could be removed if there is one entry only. And if there is more than one, it could be replace with another separator.